package.json

```diff
@@ -2,20 +2,25 @@
   "scripts": {
     "build-css": "./node_modules/.bin/postcss ./assets/main.css --use ../../node_modules/autoprefixer/lib/autoprefixer.js -o main.css",
     "build": "tsc && npm run build-css",
+    "build-dist-js": "yarn run build-tsickle && yarn run closure-compile",
+    "closure-compile": "cat index.js > tmp.js && cat tmp.js | java -jar ./node_modules/google-closure-compiler/compiler.jar --language_in=ECMASCRIPT6 --language_out=ECMASCRIPT5 --externs=externs.js --compilation_level ADVANCED_OPTIMIZATIONS --rewrite_polyfills=false | tee > index.js",
+    "build-tsickle": "tsickle --externs=externs.js -- -p .",
     "watch": "tsc -w",
     "watch-css": "yarn build-css -- -w",
     "start": "http-server",
     "now-start": "http-server -p 80"
   },
   "browserslist": [
-     "> 5%"
+    "> 5%"
   ],
   "dependencies": {
     "typescript": "^2.3.3"
   },
   "devDependencies": {
     "autoprefixer": "^7.1.1",
+    "google-closure-compiler": "^20170521.0.0",
```

Comment:
Any specific reason you're not using [`google-closure-compiler-js`](https://www.npmjs.com/package/google-closure-compiler-js) instead? (that doesn't require Java)